src/js/components/SavedSpacesList.js

```diff
@@ -52,8 +52,8 @@ export default function SavedSpacesList({spaces, spaceContextMenu}: Props) {
                 !s.ingesting() &&
                   showContextMenu(spaceContextMenu(s.id, s.name))
               }}
-              className={`space-link ${
```

Comment:
Oh yeah! I've been bit by this before. If you feel like using it, the `classNames` library is nice for this. The code you wrote would be equivalent to: 
```js
const className = classNames("space-link", {
  "current-space-link": s.id === currentSpaceId
})
```
 
The function accepts any number of aguments. If it's given a string, that gets added to the className. If it's given an object, all of the keys that have a truthy value to them will be added to the className. If it's given anything falsey, those are excluded.

Pretty handy, and avoids the "extra-space in the className" issue.
```